src/createCachedSelector.js

```diff
@@ -24,7 +24,7 @@ function createCachedSelector(...funcs) {
     };
     funcs.push(resultFuncWithRecomputations);
 
-    const cache = options.cacheObject || new defaultCacheCreator();
+    const cache = options.cacheObject || new defaultCacheCreator({ cacheSize: 8 });
```

Comment:
Why 8?

Reply:
Just a guesstimate as a first pass, I think we might also go higher actually – perhaps something like ~20-30 to account for situations where perhaps there is a list on screen and each item is connected with different params?

Reply:
Any further thoughts on a good starting number? Otherwise I'm going to up this to `20` and give that a go.

Reply:
What is the number, kilobytes? megabytes? is there any data we could use to make an informed decision?

Reply:
@rickharris I think it's object count. The [sdk has a similar option](https://sdk.goabstract.com/docs/configuration/#maxcachesize) and I had the same question. With the sdk we went with a default of `0` because we didn't want to enable the cache by default

@tommoor Without much data to backup the cache's memory usage is, my recommendation would be to side on a larger number to avoid any performance penalties. My concern is unnecessarily cleaning the cache and to account for the case where we have lots of cached objects in demand like with infinite pagination. Maybe `100` but really that could be too little also?

Could adjust this PR to add only minimum hook to inject the `defaultCacheCreator` so that way we can explore this value in our app without having to release this fork each time

Reply:
Ohh ok I think I vastly misunderstood. So is this number per-cached-selector or global? Since `8` would be a super low global number, I'm guessing it's per-selector.

Reply:
It's per-selector yep.

I injected some logging into `re-reselect` to see what kind of numbers we're getting up to – for most selectors the number stays low `< 10`.

On the projects listing screen there is one cached selector per project so it scales with that upto around `50` for my own account. 

The unbounded ones that this is accounting for is things like commits where we want to purge selectors for commits that you might not have looked at for a while. E.g something like navigating through a layerset or back through commit history will increase selector caches by one for every layer / commit.

Really that project listing screen becomes something of a limiting factor because whilst it's unvirtualized anything less than the total projects will cause cache churn.

Reply:
Looking specifically at the selectors in question on ProjectListItem they all return either bools or numbers so it's not a big deal. 50 seems like a more than reasonable number based on this/